content/en/blog/_posts/2021-04-22-gateway-api/2021-04-22-gateway-api.md

```diff
@@ -0,0 +1,199 @@
+3. It’s possible to provide extensibility for more complex capabilities in a way that does not sacrifice the user experience of the core API
+
+
+## Introducing the Gateway API
+
+This led to design principles that allow the Gateway API to improve upon Ingress:
+
+- **Expressiveness** - In addition to HTTP host/path matching and TLS, Gateway API can express capabilities like HTTP header manipulation, traffic weighting & mirroring, TCP/UDP routing, and other capabilities that were only possible in Ingress through custom annotations.
+- **Role-oriented design** - The API resource model reflects the separation of responsibilities that is common in routing and Kubernetes service networking.
+- **Extensibility** - The resources allow arbitrary configuration attachment at various layers within the API. This makes granular customization possible at the most appropriate places.
+- **Flexible conformance** - This is where the Gateway API aims to improve the portability Ingress through more [flexible conformance](https://gateway-api.sigs.k8s.io/concepts/guidelines/#conformance). With core (mandatory support), extended (portable if supported), and custom (no portability guarantee) conformance levels, the API is explicit about its portability without constraining implementations-specific capabilities.
+
+
+### What does the Gateway API look like?
+
+The Gateway API introduces a few new resource types:
+
+- **GatewayClasses** are a cluster-scoped resource that act as templates to explicitly define behavior for Gateways derived from them. This is similar in concept to StorageClasses, but for Gateways, our Ingress-like object.
+- **Gateways** are the deployed instances of GatewayClasses. They are the logical representation of the data-plane which performs routing, which may be in-cluster proxies, hardware LBs, or cloud LBs.
+- **HTTPRoutes** are the matching, filtering, and routing rules that get applied to Gateways, which results in the configuration of the underlying data-plane. There are also other kinds of Routes that exist too (since they are protocol-specific).
+- **xRoutes** are not a resource itself, but represent that Routes are protocol-specific. This provides a path for future evolution.  TLSRoutes, TCPRoutes, and UDPRoutes already have proposals with the future possibility of other protocols such as GRPCRoutes.
+
+![gateway-api-resources](./gateway-api-resources.png)
+
+### Gateway Controller Implementations
+
+The good news is that although Gateway is in [v1Alpha1](https://github.com/kubernetes-sigs/gateway-api/releases), there are already several [Gateway controller implementations](https://gateway-api.sigs.k8s.io/references/implementations/) that you can run. Since it’s a standardized spec, the following example could be run on any of them and should function the exact same way. Check out [getting started](https://gateway-api.sigs.k8s.io/guides/getting-started/) to see how to install and use one of these Gateway controllers.
```

Comment:
nit:
```suggestion
The good news is that although Gateway is in [v1alpha1](https://github.com/kubernetes-sigs/gateway-api/releases), there are already several [Gateway controller implementations](https://gateway-api.sigs.k8s.io/references/implementations/) that you can run. Since it’s a standardized spec, the following example could be run on any of them and should function the exact same way. Check out [getting started](https://gateway-api.sigs.k8s.io/guides/getting-started/) to see how to install and use one of these Gateway controllers.
```

Reply:
I'd just say it's in alpha. If the API version were v1alpha2 it'd still be in alpha.

Reply:
+1 to just "is in alpha".